### PR TITLE
Update build-windows-toolchain.bat

### PR DESCRIPTION
**Pull request by compnerd**
Update the toolchain build rules to allow us to package the VCRuntime dependency into the installer.  This avoids the VS dependency in the case that we are cross-compiling to a non-Windows platform.

### CODE REVIEW

**Comment by compnerd** [comment]
Please build with following PRs:
https://github.com/apple/swift-installer-scripts/pull/251

@swift-ci please build toolchain Windows platform

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

File: utils/build-windows-toolchain.bat [hunk 1]
```diff
@@ -973,6 +973,8 @@ msbuild %SourceRoot%\swift-installer-scripts\platforms\Windows\bundle\installer.
   -p:BaseReleaseDownloadUrl=https://download.swift.org/development/windows ^
   -p:Configuration=Release ^
   -p:BaseOutputPath=%PackageRoot%\online\ ^
+  -p:VCRedistInsaler="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
```
Comment (MarSe32m) [inline utils/build-windows-toolchain.bat:976]: (Random drive-by typo catch 😁)
```suggestion
  -p:VCRedistInstaller="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
```

Comment (compnerd) [reply utils/build-windows-toolchain.bat:976]: 😒 thank you @MarSe32m!

File: utils/build-windows-toolchain.bat [hunk 2]
```diff
@@ -987,6 +989,8 @@ msbuild %SourceRoot%\swift-installer-scripts\platforms\Windows\bundle\installer.
   -p:BundleFlavor=offline ^
   -p:Configuration=Release ^
   -p:BaseOutputPath=%PackageRoot%\offline\ ^
+  -p:VCRedistInsaler="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
```
Comment (compnerd) [inline utils/build-windows-toolchain.bat:992]:
```suggestion
  -p:VCRedistInstaller="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
```

**Comment by compnerd** [comment]
Please build with following PRs:
https://github.com/apple/swift-installer-scripts/pull/251

@swift-ci please build toolchain Windows platform

**Comment by compnerd** [comment]
Please build with following PRs:
https://github.com/apple/swift-installer-scripts/pull/251

@swift-ci please test

**Review by shahmishal — APPROVED** [review]
